### PR TITLE
Standardize testgrid dashboard prefixes

### PR DESCRIPTION
**Pull request by fejta**
Require all dashboards and dashboard groups to begin with a known company name, sig, wg or presubmits. Add unit test to enforce this.

Move all dashboards into a sig-foo, presubmit-foo, company dashboard/dashboard group.

Add presubmits-test-infra dashboard.

/assign @michelle192837 @krzyzacy

### CODE REVIEW

**Comment by fejta** [comment]
/hold

Until the next testgrid release is live

File: testgrid/config/config_test.go [hunk 1]
```diff
@@ -183,6 +214,22 @@ func TestConfig(t *testing.T) {
...
+					found = true
+					break
+				}
+			}
+			if found {
+				break
+			}
+		}
+		if !found {
+			t.Errorf("Dashboard %v: must prefix with one of: %v", dashboardGroup.Name, prefixes)
```
Comment (krzyzacy) [inline testgrid/config/config_test.go]: nit: s/Dashboard/DashboardGroup/

File: testgrid/config/config_test.go [hunk 2]
```diff
@@ -104,6 +119,22 @@ func TestConfig(t *testing.T) {
...
+			for _, prefix := range kind {
+				if strings.HasPrefix(dashboard.Name, prefix+"-") || dashboard.Name == prefix {
+					found = true
+					break
+				}
+			}
+			if found {
+				break
+			}
+		}
```
Comment (krzyzacy) [inline testgrid/config/config_test.go]: the prefix check probably can be in it's own function, as dashboard and dashboardGroup shares the same logic

**Comment by krzyzacy** [comment]
/lgtm
some bikeshedding

File: testgrid/config/config_test.go [hunk 3]
```diff
@@ -183,6 +214,22 @@ func TestConfig(t *testing.T) {
...
+		if !found {
+			t.Errorf("Dashboard group %v: must prefix with one of: %v", dashboardGroup.Name, prefixes)
+		}
+
 		// All dashboardgroup must not have duplicated names
```
Comment (michelle192837) [inline testgrid/config/config_test.go]: Can this enforce two additional/stronger constraints?

1) No dashboard group OR dashboard can have the same name
2) No dashboard can be in more than one group.

Comment (fejta) [reply testgrid/config/config_test.go]: 1. Done
2. enforced by L243

File: testgrid/config/config_test.go (hunk 2 above)
Comment (fejta) [reply testgrid/config/config_test.go]: Meh

**Pull request merged**